src/instance.rs

```diff
@@ -336,18 +336,14 @@ impl<'py> Bound<'py, PyAny> {
     ///
     /// # Safety
     ///
-    /// - `ptr` must be a valid pointer to a Python object
+    /// - `ptr` must be a valid pointer to a Python object (or null, which will cause a panic)
```

Comment:
It might make sense to (also?) put this in a "Panics" heading as is done for the `Py` variants.

Reply:
Good idea, done 👍